src/template.ts

```
@@ -1,40 +1,33 @@
-  {
+  } as const
+
+  static readonly 'rollup' = {
     name: 'rollup',
     template: 'rollup-template-sky',
-    color: kleur.blue,
-  },
-]
+    color: kleur.magenta,
+  } as const
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)